doc/Overview/Reactive/How-tos/CreateMvuxProject.md

```diff
@@ -0,0 +1,39 @@
+---
+uid: Overview.Reactive.HowTos.CreateMvuxProject
+---
+
+# How to set up an MVUX project
+
+In this tutorial you'll learn how to set up an Uno Platform project ready to use with the MVUX architecture and its tools.
+
+### Creating an Uno Platform project
+
+1. Make sure your environment is set up properly by using [uno check](external/uno.check/doc/using-uno-check.md).
+1. Create a new Uno App by following steps 1-5 in [this](https://platform.uno/docs/articles/getting-started-tutorial-1.html) tutorial
+when using the [Visual Studio Uno templates](https://platform.uno/docs/articles/get-started-vs-2022.html#install-the-solution-templates),
+or by running the following command, using an appropriate name (`MyAppName` in this page).
+
+  ```cmd
+  dotnet new unoapp-uwp -o MyAppName
```

Comment:
```suggestion
  dotnet new unoapp -o MyAppName
```

Reply:
If you use the default unoapp template, reactive is already wired up

If you want a blank template, then use
dotnet new unoapp -preset blank